### ISSUE TITLE
Relicense to GPLv2 or later

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#2276]
While relicensing mutagen I've also included contributions to quodlibet, so many of the original authors are covered as well: https://github.com/quodlibet/relicensing

Joe agreed to relicense the distutils code to standard MIT, which happened in 5957ba8edb2a03f96fa0fd88dce00e9665709b87

The plan I have in mind is going file by file, check out the history and rewrite the license header.
Once the easy stuff is done we can email the rest.

We don't need the license change right away, so there is no deadline.

```
git blame -C -C -M -e
git shortlog -sne
```

**Comment by lazka** [comment]
@CreamyCookie @elfalem @Meriipu

Since I don't have emails of you three, does anyone of you agree to provide your existing contributions to this repo under the terms of the GPLv2 or later (instead of GPLV2 only)?

**Comment by lazka** [comment]
Feel free to send a mail to reiter.christoph@gmail.com if you have any questions.

**Comment by Meriipu** [comment]
> does anyone of you agree to provide your existing contributions to this repo under the terms of the GPLv2 or later (instead of GPLV2 only)?

I would provide the contributions public domain insofar as that is possible so I suppose, sure.

**Comment by elfalem** [comment]
@lazka I agree to GPLv2 or later (or any other open source license).

**Comment by lazka** [comment]
@Meriipu @elfalem thanks!

**Comment by CreamyCookie** [comment]
@lazka I also agree to provide my existing contributions to this repository under the terms of the GPLv2 or later. :+1:

**Comment by lazka** [comment]
@CreamyCookie thanks!
